src/sympc/tensor/mpc_tensor.py

```diff
@@ -368,39 +366,22 @@ def reconstruct(
 
         Args:
             decode (bool): True if decode using FixedPointEncoder. Defaults to True
-            get_shares (bool): True if get shares. Defaults to False.
+            get_shares (bool): Retrieve only shares.
 
         Returns:
             torch.Tensor. The secret reconstructed.
         """
+        plaintext = self.session.protocol.share_class.reconstruct(
```

Comment:
MAYBE: Could we rename this to something else? (since it might not be the plaintext result?)

Reply:
Sure, result would do.